.changeset/moody-toys-dance.md

```diff
@@ -2,8 +2,12 @@
 "@fluidframework/tree": minor
 "fluid-framework": minor
 ---
+---
+section: tree
+highlight: true
+---
 
-Allow constructing ArrayNodes from Maps and MapNodes from arrays when unambiguous.
+✨ New! ArrayNodes can now be constructed from Maps, and MapNodes from arrays when unambiguous
```

Comment:
The current wording can be easily red as "when unambiguous" only applying to the second case.
```suggestion
✨ New! When unambiguous, ArrayNodes can now be constructed from Maps and MapNodes from arrays
```